The initial work I've put into the report

wyattwayman opened the pull request:
This is the initial work I have put into the final project report. I figured I'd make a pull request before we filmed the final documentation. so far I have the introduction, part of the design overview, and a large portion of the testing sections done. Let me know if you look over it and if you have any changes you want to make.

wyattwayman committed:
initial work

wyattwayman committed:
Worked on design overview and testing sections

nhedges commented on blue_scribe.tex:
```diff
@@ -12,17 +12,100 @@
+	\item The laser brightness shall be changed for different instruction purposes.
+	\item The laser location shall be able to return a home position with a single instruction.
+	\item The microcontroller shall be able to communicate serially with the PC via USB.
+	\item The program for turning images into function commands shall ????***** I don't know how to word this 
```
The PC software shall process image files to determine the necessary laser commands to create an image similar to that found in the image file.

nhedges commented on blue_scribe.tex:
```diff
@@ -12,17 +12,100 @@
+	\item The microcontroller shall be able to receive instructions manually through a terminal on the PC.
+	\item The microcontroller shall be able to receive instructions automatically from the PC via the image breakdown program. *****
+	\item The microcontroller shall be able to send a response flag via the UART back to the PC upon instruction completion.
+	\item The microcontroller shall be able to send an error flag via the UART back to the PC upon invalid instruction.
```
the 'be able to' can be removed on some of these

nhedges commented on blue_scribe.tex:
```diff
@@ -12,17 +12,100 @@
+\end{itemize}
+\subsection{Dependencies}
+The following components are required to build the complete laser engraver design:
+\begin{itemize}
```
It would maybe be good to split these into categories- Hardware, Software, Electrical?

nhedges commented on blue_scribe.tex:
```diff
@@ -12,17 +12,100 @@
-%   \end{center}
-% \end{figure}
+
+\section*{Appendices}
```
Figures are in-line this time. Instead of the appendix, they go adjacent to the text that is most relevant.

nhedges commented on blue_scribe.tex:
```diff
@@ -12,17 +12,100 @@
+
+\subsection{Manual Instruction Input Test}
+The manual instruction input test was performed by connecting the microcontroller to a PC via a mini USB and connecting the laser and both actuators pins. Once connected, the PC sent commands serially to the microcontroller via a PuTTy terminal to ensure the microcontroller was receiving the commands and implementing the function calls correctly. We sent the following instructions: GO 5000, 5000; ...
+
```
entering commands manually might be a good thing to add to requirements (or maybe a 'capabilities' list?)

nhedges committed:
Delete blue_scribe.log

We don't need to include the log files, they are just an extra thing when you build the doc.

nhedges commented:
I would say make some changes as described, then commit and merge directly without a new pull request.
